dist/style.scss

```diff
@@ -1,685 +1,952 @@
+    font-weight: weight(regular);
+    font-size: 64px;
+    line-height: 82px;
+    color: $primary;
+  }
+  p {
+    font-family: $inter;
+    font-style: normal;
+    font-weight: weight(regular);
+    font-size: 16px;
+    line-height: 24px;
+    color: $white2;
+  }
+  button {
+    margin-top: 2rem;
+    background-color: $primary;
+    color: $base-white;
+    width: 161px;
+    height: 48px;
+    transition: all 0.3s ease-in-out;
+    &:hover {
+      background-color: $milky-white;
+      border: 1px solid $primary;
+      color: $primary;
+      transform: scale(1.1);
+    }
+  }
+}
+
+.skillset {
+  display: flex;
+  flex-direction: column;
+  .specifics {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 15px;
+    margin-top: 1.5rem;
+    h3 {
+      span {
+        color: $primary;
+        margin-right: 5px;
+        font-size: 1em;
+      }
+      font-family: $inter;
+      font-style: normal;
+      font-weight: weight(bold);
+      font-size: 30px;
+      line-height: 44px;
+      color: $base-white;
+    }
+    ul {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      li {
+        padding: 10px;
+        font-family: $inter;
+        font-style: normal;
+        font-weight: weight(regular);
+        font-size: 20px;
+        line-height: 40px;
+        color: $base-white;
+      }
+    }
+  }
+}
+
+#contact {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+}
+
+.contact-text {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 327px;
+  height: 216px;
+  margin: 6rem 1rem 3rem 2rem;
+  p {
+    font-family: $inter;
+    font-style: normal;
+    font-weight: weight(bold);
+    font-size: 32px;
+    line-height: 44px;
+    color: $base-navy;
+  }
+}
+
+.form {
+  display: none;
+}
+
+.form2 {
+  fieldset {
+    width: 328px;
+    height: 400px;
+    border: none;
+    input {
+      margin-bottom: 1rem;
+    }
+
+    input,
+    textarea {
+      padding: 10px;
+      margin-left: 1rem;
+      width: 100%;
+      border: none;
+      border-bottom: 1px solid $gray3;
+    }
+
+    input:focus,
+    textarea:focus {
+      outline: none;
+    }
+
+    textarea {
+      background-color: #e7e5e5;
+    }
+
+    button {
+      width: 122px;
+      height: 48px;
+      background-color: $primary;
+      color: $base-white;
+      border: none;
+      font-family: $roboto;
+      font-style: normal;
+      font-weight: weight(bold);
+      font-size: 17px;
+      line-height: 24px;
+      margin-left: 8em;
+      margin-top: 1rem;
+    }
+  }
+}
+
+.copyright {
+  border-top: 1px solid#DFE1E6;
+  margin-top: 8rem;
+  ul {
+    display: flex;
+    justify-content: center;
+    li {
+      margin: 15px;
+      a {
+        font-size: 1.5em;
+        color: $fade-gray;
+      }
+    }
+  }
+}
+
+@media (min-width: 450px) and (max-width: 767px) {

```

Comment:
- [ ] Please 🙏🏽 use the correct screen width asked in the project requirements to make your desktop version look like the template you selected. It will be responsive to other sizes, you don't need to specify others.

> Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider).